firmware/dev/h7dev/CMakeLists.txt

```diff
@@ -49,20 +55,20 @@ if("${TARGET}" STREQUAL "deploy")
     )
 
     embedded_binary(
-        "h7dev_app"
+        "h7dev"
         "${EMBEDDED_SRCS}"
         "${EMBEDDED_INCLUDE_DIRS}"
         "${LINKER_SCRIPT}"
         "${ARM_CORE}"
     )
-    target_link_libraries("h7dev_app.elf" "h7dev_stm32cube")
+    target_link_libraries("h7dev.elf" "h7dev_stm32cube" "littlefs_cm7")
 
-    embedded_image(
-        "h7dev"
-        "h7dev_app.hex"
-        "${CMAKE_CURRENT_BINARY_DIR}/h7dev_app.hex"
-        "h7dev_boot.hex"
-        "${CMAKE_BINARY_DIR}/firmware/boot/h7boot/h7dev_boot.hex"
-    )
+    # embedded_image(
+    #     "h7dev"
+    #     "h7dev_app.hex"
+    #     "${CMAKE_CURRENT_BINARY_DIR}/h7dev_app.hex"
+    #     "h7dev_boot.hex"
+    #     "${CMAKE_BINARY_DIR}/firmware/boot/h7boot/h7dev_boot.hex"
+    # )
```

Comment:
Now that the board is stable, could you revert disabling the bootloader? I think we should keep it on here for testing if necessary.